### PR TITLE
add location props to sample node

### PR DESCRIPTION
**Pull request by wangfan860**
### Improvements
Add location related props in sample node:
`country_region`
`province_state`
`city`
`county`
`continent`

Related discusion: https://cdis.slack.com/archives/C01FS3JNLQ7/p1607104233015100

### CODE REVIEW

**Review by svburke — APPROVED** [review]

File: gdcdictionary/schemas/sample.yaml [hunk 1]
```diff
@@ -728,6 +422,26 @@ properties:
     enum:
       - host-associated
 
+# Prop originated from ncbi-covid-19 project
+  country_region:
+    description: "Name of the country or region where the sample was collected."
+    type: string
+
+# Prop originated from ncbi-covid-19 project
+  city_province:
```
Comment (gkuffel) [inline gdcdictionary/schemas/sample.yaml:431]: This prop should be "province_state"

File: gdcdictionary/schemas/sample.yaml [hunk 2]
```diff
@@ -728,6 +422,26 @@ properties:
...
+    description: "Name of the city or province where the sample was collected."
```
Comment (gkuffel) [inline gdcdictionary/schemas/sample.yaml:432]: There should be a prop for just "city"

**Review by gkuffel — COMMENTED** [review]
Please make the indicated changes.

File: gdcdictionary/schemas/sample.yaml (hunk 1 above)
Comment (wangfan860) [reply gdcdictionary/schemas/sample.yaml]: property name and description was changed

File: gdcdictionary/schemas/sample.yaml (hunk 2 above)
Comment (wangfan860) [reply gdcdictionary/schemas/sample.yaml]: city added

**Review by svburke — DISMISSED** [review]

**Review by gkuffel — APPROVED** [review]

**Review by gkuffel — APPROVED** [review]

**Review by svburke — APPROVED** [review]

**Pull request merged**